beeai/Containerfile.c10s

```diff
@@ -1,26 +1,40 @@
-FROM fedora:42
+FROM quay.io/centos/centos:stream10
 
-# Install system dependencies
-RUN dnf -y install \
+RUN dnf -y install epel-release \
+    && dnf config-manager --set-enabled crb \
+    && dnf -y update
+
+RUN dnf -y install --allowerasing \
       centpkg \
       curl \
       gcc \
       git \
       python3 \
       python3-pip \
-      python3-redis \
-      python3-specfile \
-      rpmbuild \
+      python3-devel \
+      python3-rpm \
+      rpm-build \
       rpmdevtools \
       rpmlint \
-      spectool \
-    && dnf -y clean all
+      autoconf \
+      automake \
```

Comment:
could you please add cmake and meson; those 2 are also pretty common build systems

Reply:
sure, I just noticed https://github.com/packit/jotnar/issues/172, will also reference it